workers/unity/Packages/io.improbable.gdk.tools/GdkToolsConfigurationWindow.cs

```diff
@@ -18,6 +19,7 @@ public class GdkToolsConfigurationWindow : EditorWindow
         private const string DevAuthTokenSectionLabel = "Dev Auth Token Settings";
         private const string DevAuthTokenLifetimeLabel = "Token lifetime (days)";
         private const string CodeGeneratorLabel = "Code generator";
+        private const string CustomSnapshotPathLabel = "Custom snapshot path";
```

Comment:
```suggestion
        private const string CustomSnapshotPathLabel = "Selected snapshot path";
```

Maybe? Strictly speaking, all snapshots are custom 😉

Reply:
And this can be the "default" snapshot as well